src/components/FormContainer.jsx

```diff
@@ -7,33 +7,36 @@ const selectTab = (currentTab, tabList)  => {
 };
 
 const FormContainer = props => {
-
+	
 	const tabList = [
-		{description: props.tabNames.map(name => name.Tab1), key: 'ServiceRequestForm', value: 0, shouldDisableForm: false, requiresLocation: true},
-		{description: props.tabNames.map(name => name.Tab2), key: 'ProvideDetails', value: 1, shouldDisableForm: props.shouldDisableForm, requiresLocation: true},
-		{description: props.tabNames.map(name => name.Tab3), key: 'AdditionalInformation', value: 2, shouldDisableForm: props.shouldDisableForm, requiresLocation: props.requiresLocation},
-		{description: props.tabNames.map(name => name.Tab4), key: 'Blank', value: 3, shouldDisableForm: false, requiresLocation: true},
+		{description: props.tabNames.map(name => name.Tab1), id: 0,  key: 'ServiceRequestForm', shouldDisableForm: false, isAccela: true},
+		{description: props.tabNames.map(name => name.Tab2), id: 1, key: 'ProvideDetails', shouldDisableForm: props.shouldDisableForm, isAccela: true},
```

Comment:
i think we should probably avoid namign this after a vendor product, in the case that this changes to another vendor product. Maybe something like `isPanelRequired` maybe?

Reply:
naughty space messed me up. This has been changed